src/Styleguide/Sections/__stories__/OtherAuctions.story.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { OtherAuctions } from "../OtherAuctions"
+import { Section } from "../../Utils/Section"
+import { AuctionCard } from "../../Components/AuctionCard"
+
+const auctions = [
+  {
+    src: "https://picsum.photos/200/180/?random",
+    headline: "Sotheby’s",
+    subHeadline: "Contemporary Day Sale",
+    badge: "In progress",
+  },
+  {
+    src: "https://picsum.photos/200/180/?random",
+    headline: "Sotheby’s",
+    subHeadline: "Contemporary Day Sale",
+    badge: "In progress",
+  },
+  {
+    src: "https://picsum.photos/200/180/?random",
+    headline: "Sotheby’s",
+    subHeadline: "Contemporary Day Sale",
+    badge: "In progress",
+  },
+  {
+    src: "https://picsum.photos/200/180/?random",
+    headline: "Sotheby’s",
+    subHeadline: "Contemporary Day Sale",
+    badge: "In progress",
+  },
+]
+
+storiesOf("Styleguide/Sections", module).add("Other Auctions", () => {
+  return (
+    <Section title="Responsive Other Auctions">
+      <OtherAuctions>
+        {auctions.map(auction => <AuctionCard {...auction} />)}
+      </OtherAuctions>
+    </Section>
```

Comment:
Will `LargeOtherAuctions` and `SmallOtherAuctions` eventually be shown here?

Reply:
Could be... that component is already responsive. It's just relying on that.